medmodels/statistic_evaluations/evaluate_compare/evaluate.pyi

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Dict, List, Optional, Tuple, Union
+
+from medmodels.medrecord.medrecord import MedRecord
+from medmodels.medrecord.querying import NodeOperation
+from medmodels.medrecord.schema import AttributeType
+from medmodels.medrecord.types import (
+    AttributeSummary,
+    Group,
+    GroupInputList,
+    MedRecordAttribute,
+    NodeIndex,
+)
+
+class CohortEvaluator:
+    """Configuration for a cohort for evaluation and comparison.
+
+    Needs a medrecord and the corresponding patient group that should form the cohort.
+    The cohort group can be a predefined group in the MedRecord or a node query.
+
+    """
+
+    medrecord: MedRecord
+    name: str
+    cohort_group: Group
+    time_attribute: MedRecordAttribute
+    attributes: Optional[Dict[str, MedRecordAttribute]]
+    concepts_groups: Optional[GroupInputList]
+    attribute_summary: Dict[Group, AttributeSummary]
+    attribute_types: Dict[MedRecordAttribute, AttributeType]
+
+    def __init__(
+        self,
+        medrecord: MedRecord,
+        name: str,
+        cohort_group: Union[Group, NodeOperation] = "patients",
+        time_attribute: MedRecordAttribute = "time",
+        attributes: Optional[Dict[str, MedRecordAttribute]] = None,
+        concepts_groups: Optional[GroupInputList] = None,
+    ) -> None: ...
+    def get_concept_counts(
+        self,
+    ) -> List[Tuple[NodeIndex, int]]: ...
+    def get_top_k_concepts(
+        self,
+        top_k: int,
+    ) -> List[NodeIndex]: ...
```

Comment:
Maybe it would nice here to have a way to know how many times each one of them is represented. Just a mere list does not know whether the first is 10,000 times, second is only 20 for instance. Maybe a specific class like the AttributeSummary that when represented, makes a table with the node index and the amount of edges for that concept would be a nice touch.

Reply:
There is also a concept counts function :)

Reply:
How they will be represented in the table, I'll figure out then honestly